packaging/package.sh

```diff
@@ -48,6 +48,13 @@ function packaging() {
     fi
 
     DISTRO=$3
+
+    JDK=$(rpm -qa | grep "java-1...0-openjdk-devel")
+    for version in $JDK; do [ "${version:7:1}" -ge "7" ] && echo  ${version:7:1}; done;
+    if [ "$?" -gt "0" ] || [ -z "$JDK" ] ; then
```

Comment:
Yes.. definitely, I didn't test jdk6 silly me... I'll fix that when I can
get my pc back up because I'm currently driving across Europe :-). Just
remove that check from package.sh to test the rest, just punched that in to
remove constraints for java 7 and 8 from the spec files, and still inform
the user if he is missing jdk
On May 13, 2015 11:52 PM, "Erik Weber" <notifications@github.com> wrote:

> In packaging/package.sh
> <https://github.com/apache/cloudstack/pull/238#discussion_r30280222>:
>
> > @@ -48,6 +48,13 @@ function packaging() {
> >      fi
> >
> >      DISTRO=$3
> > +
> > +    JDK=$(rpm -qa | grep "java-1...0-openjdk-devel")
> > +    for version in $JDK; do [ "${version:7:1}" -ge "7" ] && echo  ${version:7:1}; done;
> > +    if [ "$?" -gt "0" ] || [ -z "$JDK" ] ; then
>
> My previous comment is most likely related to this. If the last grep line
> doesn't match -ge 7, it'll fail
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/cloudstack/pull/238/files#r30280222>.
>